openapi-integration-tests/src/test/java/io/ballerina/openapi/idl/client/IDLClientGenPluginNegativeTests.java

```diff
@@ -42,41 +47,59 @@ public class IDLClientGenPluginNegativeTests extends OpenAPITest {
     public static final Path TEST_RESOURCE = Paths.get(RESOURCES_PATH.toString() + "/client-idl-projects");
-
     @BeforeClass
     public void setupDistributions() throws IOException {
         TestUtil.cleanDistribution();
     }
 
     @Test(description = "Here uses the graphQl yaml as non openapi client")
     public void testNonOpenAPIClientDeclaration() throws IOException, InterruptedException {
-        boolean successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_07"),
+        Process successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_07"),
                 new ArrayList<>());
         File dir = new File(RESOURCE.resolve("client-idl-projects/project_07/generated/").toString());
         Assert.assertFalse(dir.exists());
+        String msg = " ERROR [main.bal:(1:1,1:176)] no matching plugin found for client declaration";
+        logComparison(successful, msg);
     }
 
     @Test
     public void testInvalidSwaggerRemotePath() throws IOException, InterruptedException {
-        boolean successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_10"),
+        Process successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_10"),
                 new ArrayList<>());
         File dir = new File(RESOURCE.resolve("client-idl-projects/project_10/generated/").toString());
         Assert.assertFalse(dir.exists());
+        String msg = "ERROR [main.bal:(2:5,3:13)] unable to get resource from uri, reason: ";
+        logComparison(successful, msg);
     }
 
     @Test
     public void testInvalidSwaggerLocalPath() throws IOException, InterruptedException {
-        boolean successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_11"),
+        Process successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_11"),
                 new ArrayList<>());
         File dir = new File(RESOURCE.resolve("client-idl-projects/project_11/generated/").toString());
         Assert.assertFalse(dir.exists());
+        String msg = "ERROR [main.bal:(2:5,2:35)] unable to get resource from uri, reason: no protocol: openapi2.yaml";
+        logComparison(successful, msg);
     }
 
     @Test
     public void testInvalidSwaggerContract() throws IOException, InterruptedException {
-        boolean successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_12"),
+        Process successful = executeRun(DISTRIBUTION_FILE_NAME, TEST_RESOURCE.resolve("project_12"),
                 new ArrayList<>());
         File dir = new File(RESOURCE.resolve("client-idl-projects/project_12/generated/").toString());
         Assert.assertFalse(dir.exists());
+        String msg = " ERROR [main.bal:(2:5,2:34)] OpenAPI definition has errors:";
+        logComparison(successful, msg);
+    }
+
+    private static void logComparison(Process successful, String msg) throws IOException {
```

Comment:
`logComparison` is not a valid method name by the convention. Please use a proper name 
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Methods%20should%20be%20verbs%2C%20in,of%20each%20internal%20word%20capitalized.&text=Except%20for%20variables%2C%20all%20instance,words%20start%20with%20capital%20letters.